### DISCUSSION TITLE
Welcome to INCF-Impact-visualization-Portal Discussions!

### DISCUSSION THREAD

**Discussion by atharva434** [atharva434/INCF-Impact-visualization-Portal#5] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sajontahsen** [comment]
Hello @atharva434! I'm keen on contributing to this project under GSoC'24 and have a few queries:

1. **Graphical Representation of Impact:**  After reviewing your documentation (which was incredibly helpful for getting acquainted with the project), it seems that enhancing the Dynamic Visualization of Impact Metrics is a key next step for the project. I'm eager to tackle this challenge. Could you suggest how to begin or any specific metrics that need highlighting?
2. **Expanding Langchain Integration for New Features:** Exploring the codebase, I saw Cohere used with LangChain for chatbot functionality. This led me to consider extending LangChain for Sentiment Analysis and Semantic Search. Could we discuss the specific goals and requirements for this year to refine my approach?

I admire the progress made so far and am eager to collaborate on this meaningful project. I believe I can add value here, as this aligns well with my academic and profession experiences. Looking forward to your guidance on this.

About me, I am a recent CS graduate actively engaged in machine learning research. One of my recent highlights was having my work on multi-teacher knowledge distillation for leukemia classification showcased at the NewInML workshop at NeurIPS 2023. In my current role as an ML Engineer, I work on projects related to computer vision, time series forecasting, and data analysis. My resume, linked below, highlights my previous projects and research publications relevant to this initiative. 

Best regards,
Tahsen Islam Sajon
[Resume](https://drive.google.com/file/d/1nUR4WYFnh4pY0yUSeteKR8PqZHS_9xgh/view?usp=drivesdk) | [Email](mailto:sajon.tahsen@gmail.com) | [LinkedIn](https://www.linkedin.com/in/tahsen-islam-sajon)
